### DISCUSSION TITLE
Unrestricted password?

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#5366] [Q&A]
Dear Sirs,
I respect the security concerns but in my case I have a "strange" requirement: to have RustDesk passwords contained only numbers (or passwords that does not respect the default requirements such as: length 8 or more characters & digit & Uppercase & lowercase).
Is there any easy way to do it without the need to compile a special modified version ?

Thanks and Best Regards,
Mike Kranidis

**Comment by Loremipsumdolorsitametconsetetur** [comment]
@rustdesk I’m a RustDesk PRO user and would also appreciate an optional setting for my custom client generator to restrict OTPs to numeric-only codes (digits only, no letters). This would simplify input, especially for users on the phone where letters are sometimes hard to understand. A toggle for this option would be really helpful!

**Reply by rustdesk** [reply]
Do you mean here, OTP?

![image](https://github.com/user-attachments/assets/652b15dd-9bf0-4d7a-bc42-f5462e1f37a0)

**Reply by Loremipsumdolorsitametconsetetur** [reply]
yes, exactly!

**Reply by rustdesk** [reply]
@fufesou add this advanced option, call it "numeric-one-time-password".

**Reply by rustdesk** [reply]
Add below "length" line, also add to strategy.

![image](https://github.com/user-attachments/assets/0e1ddcd7-c18d-4c04-b817-1aa49a6f24bd)

**Reply by Loremipsumdolorsitametconsetetur** [reply]
Thank you for taking a look at this - I am amazed by the speed of your replies!

**Reply by rustdesk** [reply]
We don't always respond so quickly, only when your request resonates with us, we will respond quickly. :(
